docs/uni1.html

```
@@ -0,0 +1,445 @@
+<title>Universe Pipeline Ledger</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)